Hbase docs - cleanup (#521)
stackabletech/hbase-operator: docs/modules/hbase/pages/usage-guide/operations/index.adoc

This section of the documentation is intended for the operations teams that maintain a Stackable Data Platform installation.

> Please read the xref:concepts:operations/index.adoc[concepts page on operations] that contains the necessary details to operate the platform in a production environment.

Reviewer: ```suggestion
See the xref:concepts:operations/index.adoc[concepts page on operations] for more details on production environment operations.
```
